tsinfer/formats.py

```diff
@@ -305,7 +305,7 @@ def zarr_summary(array):
     return ret
 
 
-def chunk_iterator(array, indexes=None, mask=None, dimension=0):
+def chunk_iterator(array, indexes=None, mask=None, orthogonal_select=None, dimension=0):
```

Comment:
mask should be ``select`` here, it's being used in the wrong sense currently.

Reply:
Fixed in https://github.com/tskit-dev/tsinfer/pull/896/commits/3fdf3b9b1df88a09a6506d015e34b7dce13ab49d